terraform/modules/stack/messaging.tf

```diff
@@ -398,6 +398,29 @@ module "registered_bag_notifications_queue" {
   dlq_alarm_arn = var.dlq_alarm_arn
 }
 
+# bag indexer
+
+module "bag_indexer_input_queue" {
```

Comment:
I think this will orphan the other queue this used to use? Merge master & remove it?